features/user_can_edit_post.feature

```diff
@@ -0,0 +1,37 @@
+Feature: User can edit post
+"As a visitor,
+In order to change my posts,
+I would like the ability to edit them"
+  
+  Background:
+    Given the following user exists
+    | name    |
+    | Fat Bob |
+    | Slim Jim|
+    And the following post exists
+    | title        | text          | user     |
+    | Codeacademy  | Smooth-ride   | Fat Bob  |
+    
+  Scenario: Reader can edit his own posts
+    When I am logged in as "Fat Bob"  
+    And I visit the site
+    And I click "FLORUM"
+    And I click on "Open" 
+    Then I should see "Author: Fat Bob"
```

Comment:
You dont have to test for this in this feature

Reply:
I have to click Open do be able to Edit the post! 
Or do u mean its useless to have "Then I should see" ??